code/turf/turf_autoalign.dm

```diff
@@ -669,6 +669,46 @@
 		..()
 		setMaterial(getMaterial("coral"))
 
+// lead wall resprite by skeletonman0.... hooray for smoothwalls!
+/turf/unsimulated/wall/auto/leadblue
```

Comment:
I'd rather these share an abstract ancestor, this is a lot of needless duplication.

Reply:
fixed